src/caf/core/data_structures.py

```diff
@@ -460,6 +479,16 @@ def translate_zoning(
             The weighting to use when building the translation. Must be
             one of TranslationWeighting.
 
+        check_totals: bool = True
+            Whether to raise a warning if the translated total doesn't match the
+            input total. Should be set to False for one-to-one translations.
```

Comment:
I think this is badly phrased, what I meant was it should be set to zero where you're using a translation of all 1s to disaggregate factors for example, so that the output would be larger than the input by design. That's a short term fix though, as I think a.) check_totals shouldn't raise an error, and b.) that option for disaggregating factors should be an explicit option rather than a workaround.